eo-runtime/src/test/java/EOorg/EOeolang/RamTest.java

```diff
@@ -46,8 +53,8 @@
  * @checkstyle ParameterNumberCheck (17 lines)
  */
 @Execution(ExecutionMode.CONCURRENT)
+@DisabledOnOs(OS.MAC)
```

Comment:
Isn't it possible to make the test pass under macOS too?

Reply:
@OlesiaSub 
It probably does pass on macos. but from the GHA logs it seems that on macos JVM uses one thread here even though multiple threads are available. 
I don't have macos to debug this issue. 
I use Arch, btw

Reply:
@andreoss 
Ah ok I see. 
Maybe there is somebody who could run this on macos just to confirm that everything works fine? Or do you think it is unnecessary?

Reply:
@OlesiaSub 
If `Ram` doesn't work on macos, it would be a new bug. Here the GHA environment for macos is just insufficient to run a concurrent test.